src/pages/sighting/AssetGroupSighting.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect } from 'react';
+import { useParams, useHistory } from 'react-router-dom';
+
+import SightingCore from './SightingCore';
+import useAssetGroupSighting from '../../models/assetGroup/useAssetGroupSighting';
+
+export default function AssetGroupSighting() {
+  const history = useHistory();
+  const { id } = useParams();
+
+  const {
+    data: assetGroupSightingData,
+    loading: assetGroupLoading,
+    error: assetGroupError,
+    statusCode: assetGroupStatusCode,
+  } = useAssetGroupSighting(id);
+
+  const sightingId = assetGroupSightingData?.sightingGuid;
+
+  useEffect(
+    () => {
+      if (sightingId) {
+        history.push(`/sightings/${sightingId}`);
```

Comment:
I'm a little confused. Does this actually do the redirect or just push it into an array that we can check later and navigate away based on presence/absence in history?

Reply:
If the latter, where is the logic that is actually changing the route?

Reply:
Sorry i missed this one. it does the redirect, the name is just weird. check out the react-router docs for more info